tests/javascripts/enhancedTextbox.test.js

```diff
@@ -289,6 +287,35 @@ describe('Enhanced textbox', () => {
 
     });
 
+    test("If a resize changes the textarea's dimensions, the width of the element below should still match", () => {
```

Comment:
To match what you did on line 257.
```suggestion
    test("If a resize changes the textarea's width, the width of the element below should still match", () => {
```